### ISSUE TITLE
Project 2 Issues

### ISSUE THREAD

**Issue by Tientuine** [xYukoNx/IntroProgrammingProjectAtrocitus#6]
Address the following issues in your Project2 branch, then merge into master before starting on Project3.

- You need more commits. -2
- Your commits should be spaced out over time! -1
- Movement is broken in several situations (some moves crash the game)! -4
- Indentation is not consistent and too large/extreme! -2
- Spacing in/around operators and parentheses is not consistent. -1
- Too many "press enter" prompts, I have a hard time keeping track of what's happening -1
- The `inTritle` function might be overkill - a function that calls just two other little functions isn't really needed nor any easier to read. -0
- Avoid capitalizing variable/function names (except constructors, which we aren't using yet). -0
- Why does player get points for pressing Enter? -0
- The overall design is very confusing... consider rethinking this in the next version. -0
